pkg/virt-launcher/virtwrap/network/selinux_context_executor.go

```diff
@@ -32,6 +32,11 @@ import (
 
 const virtHandlerSELinuxLabel = "system_u:system_r:spc_t:s0"
 
+const (
+	firstFDToSkip = 3
+	lastFDToSkip  = 256
```

Comment:
The name is not self explanatory. Skip what?
how about: `minFDToCloseOnExec` and `maxFDToCloseOnExec` ?